### ISSUE TITLE
Issues with bkper.com and mobile browsers

### ISSUE THREAD

**Issue by alessandropetrone** [bkper/bkper-issues#49]
When I open bkper.com on Chrome for Android the section on the left side of the webpage stretches too far on the right, occupying at least two-thirds of the screen and making almost impossible to read the content of the section on the right side without scrolling. That didn't occur before and happens even if I request the desktop site.

My device is a Nexus 6P running Android 6.0.1; my browser's version is 50.0.2661.89.

**Comment by maelcaldas** [comment]
Hi Alessandro,

Thanks for reporting this issue. We've introduced the resize menu feature couple days ago: 

![Resize menu](https://lh3.googleusercontent.com/-2Fzb4TSZlMo/VyisXJQtc6I/AAAAAAAABkE/uecl_5C3VAQxrXzyzr-imyXXH2WDfNBrA/w530-h307-p-rw/resize_menu.gif)

You probably tapped and resized the menu on mobile and, since the menu position was saved in the local browser storage, you were not able to return the menu to original position. To avoid this, we removed the position saving, and now, even if you resize it, after you reload the page, you will be able to have it automatically returned to original position.
